frontend/src/components/google-maps/marker/LargeDeltaAreaMarkerContainer/components/ClusterMarker.tsx

```diff
@@ -0,0 +1,27 @@
+import FlexBox from '@common/FlexBox';
+import Text from '@common/Text';
+
+export interface ClusterMarkerProps {
+  count: number;
+}
+
+const ClusterMarker = ({ count }: ClusterMarkerProps) => {
+  return (
+    <FlexBox
+      p={4}
+      borderRadius="50%"
+      bgColor="#D9E5FF"
+      width="70px"
+      height="70px"
+      justifyContent="center"
+      alignItems="center"
+      css={{ border: '1px solid #3366FF' }}
```

Comment:
border 속성 없나요? border 주고 borderWidth bordercolor 지정해주면 될 것 같은데

Reply:
수정반영하였습니ㅏㄷ

Reply:
```suggestion
border
```

border={true} 생략 가능합니다!